third_party/silabs/SiWx917_sdk.gni

```diff
@@ -156,6 +155,19 @@ template("siwx917_sdk") {
       "${wifi_sdk_root}/components/device/silabs/si91x/mcu/drivers/service/power_manager/inc",
     ]
 
+    if (rgb_led) {
+      _include_dirs += [
+        "${wifi_sdk_root}/components/device/silabs/si91x/mcu/drivers/hardware_drivers/rgb_led/inc",
+        "${wifi_sdk_root}/components/device/silabs/si91x/mcu/drivers/hardware_drivers/rgb_led/config",
+      ]
+      defines += [ "SI917_DEVKIT" ]
```

Comment:
is this the correct place of this definition? I believe it should be based on the board right and not RGB_LED ?